### DISCUSSION TITLE
How can we change sidebar width?

### DISCUSSION THREAD

**Discussion by asefsoft** [filamentphp/filament#1754] [Help] [answered]
Hi
Is there a simple solution to do that?

**Comment by intrepidws** [comment, marked as answer]
You could make your own local version of [this file](https://github.com/laravel-filament/filament/blob/47c59649683f376417e7bfbf8dcf76934e7f36c2/packages/admin/resources/views/components/layouts/app/sidebar/index.blade.php) and then change `w-80` to something smaller.

**Reply by intrepidws** [reply]
You would place this new file into your app at `/resources/views/vendor/filament/components/layouts/app/sidebar/index.blade.php`

**Reply by asefsoft** [reply]
> You would place this new file into your app at `/resources/views/vendor/filament/components/layouts/app/sidebar/index.blade.php`

thanks but main area did not expand to use this new space

**Reply by intrepidws** [reply]
You'll need to make a corresponding change in [this line of this template](https://github.com/laravel-filament/filament/blob/09b4a6d5dc15d2c032aeceef3395c1a1d407eaa1/packages/admin/resources/views/components/layouts/app.blade.php#L14) to adjust the left padding.

**Reply by asefsoft** [reply]
thanks i will check that too but as i sad i wish there was a better solution.
one downside of this method is that if there will be any update to this files, i needed to manually update this files on each update of filament.

**Reply by ManissDev** [reply]
You can change it from the config file filament.php

```
'layout' => [
    'sidebar' => [
        'width' => '16rem', // Here you can change the width ( it can be px btw )
    ],
]
```
